HHS/Head-Start-TTADP
# [Prod] Capitalize tooltips, allow adding approvers to reports in "needs action" status, fix goal end date bug, fix TR permissions error

## Description of change

- Capitalize "TTA" in tooltips
- Allow adding approvers to reports in "needs action" status
- User with POC_TRAINING_REPORTS permissions who is not a TR POC should see view only of TR
- Goals on approved reports should still allow end dates to change

## How to test

- Confirm tooltips are capitalized
- Add an approver to a report that's in a needs action status and confirm flow is updated
- Impersonate a user with the POC training reports permission and confirm that you can see the view-only of TR when you click on it
- Reuse an existing goal and change an end date on an AR. The end date change should persist, not fail silently.
- Fixed a bug in `totalHrsAndRecipientGraph`.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2309
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2172
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2170
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1677
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2275
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2316

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
